src/common/ipc.ts

```diff
@@ -15,30 +18,30 @@ export async function requestMain(channel: string, ...args: any[]) {
 }
 
 async function getSubFrames(): Promise<ClusterFrameInfo[]> {
```

Comment:
Why is this function still async? Actually, to be fair why was this function ever async? The very first version of this function was as an async function in PR #1225 but even then it never had any awaits in it.

Reply:
Seems to be a leftover from the original refactoring. Original implementation had to be async but not anymore.. I'll refactor this even more to be in-line with #2046 . The slight issue here is that we need async call for in a `renderer`, sync call can be only used in `main`.